fab/fabfile.py

```diff
@@ -510,6 +510,7 @@ def _deploy_without_asking():
         execute_with_timing(db.ensure_preindex_completion)
 
         # handle static files
+        execute_with_timing(staticfiles.prime_version_static)
```

Comment:
As in, `preindex_views` exists immediately, and `ensure_preindex_completion` is basically a `wait()` that will take longer to join than `prime_version_static` will to run